lib/database/index.ts

```diff
@@ -60,7 +60,7 @@ export class PgStacDatabase extends Construct {
     });
 
     const handler = new aws_lambda.Function(this, "lambda", {
-      // defaults for configurable properties
```

Comment:
here, and the same applies to all the changes to `aws_lambda.Function` calls, the only thing we're doing is making _all_ parameters passed to `aws_lambda.Function` configurable by the user. The way we're doing this is making sure we pass `props.[insert name]LambdaFunctionOptions` in the last position. Before, some network related options (notably vpc, allowPublicSubnet) were not configurable because they were hard coded at the end. 

Although this exposes users to errors if they pass conflictual parameter values (for example, they place a lambda in a vpc and in a public subnet without the allowPublicSubnet flag), it maximizes configurability.